app/models/setting/bootdisk.rb

```diff
@@ -10,8 +10,8 @@ def self.load_defaults
       [
         self.set('bootdisk_ipxe_dir', N_('Path to directory containing iPXE images'), ipxe),
         self.set('bootdisk_syslinux_dir', N_('Path to directory containing syslinux images'), syslinux),
-        self.set('bootdisk_host_template', N_('iPXE template to use for host-specific boot disks'), 'Boot disk iPXE - host'),
-        self.set('bootdisk_generic_host_template', N_('iPXE template to use for generic host boot disks'), 'Boot disk iPXE - generic host'),
+        self.set('bootdisk_host_template', N_("iPXE template to use for host-specific boot disks"), '', 'bootdisk_host_template', nil, { :collection => Hash[ProvisioningTemplate.where(:template_kind => TemplateKind.where(:name => 'Bootdisk')).map{|temp| [temp[:name], temp[:name]]}]}),
```

Comment:
The third parameter (the default value) should stay I think, else the plugin won't be correctly configured without the templates being explicitly set.